examples/modular_llm_stack/run.py

```diff
@@ -0,0 +1,99 @@
+import json
+
+import click
+
+import data_module
+import embedding_module
+import pinecone_module  # noqa: F401
+import weaviate_module
+
+from hamilton import base, driver
+
+
+@click.command()
+@click.option(
+    "--vector_db",
+    type=click.Choice(['weaviate', 'pinecone'], case_sensitive=False),
+    default="weaviate", help="Vector database service"
+)
+@click.option(
+    "--vector_db_config",
+    prompt="Vector database config as JSON string",
+    default='{"url": "http://localhost:8080/"}',
+    help="Pass a JSON string for vector database config.\
+        Weaviate needs a dictionary {'url': ''}\
+        Pinecone needs dictionary {'environment': '', 'api_key': ''}"
+)
+@click.option(
+    "--embedding_service",
+    type=click.Choice(['openai', 'cohere', 'sentence_transformer'], case_sensitive=False),
+    default="sentence_transformer", help="Text embedding service."
+)
+@click.option(
+    "--embedding_service_api_key", 
+    prompt=True,
+    default=None,
+    help='API Key for embedding service. Needed if using OpenAI or Cohere.'
+)
+@click.option(
+    "--model_name", 
+    default=None,
+    help='Text embedding model name.'
+)
+@click.option(
+    '--display_dag',
+    is_flag=True,
+    help="Generate a .png of the Hamilton DAG"
+)
+def main(
+    vector_db: str,
+    vector_db_config: str,
+    embedding_service: str,
+    embedding_service_api_key: str | None,
+    model_name: str,
+    display_dag: bool,
+):
```

Comment:
Can you add a cut and pasteable command for people to run with the defaults? 
E.g. 
 - run weaviate in docker
 - run this code and have it populate weaviate

Reply:
Done!